Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h

```diff
@@ -18,40 +18,15 @@ namespace JS {
     P(__defineSetter__)                      \
     P(__lookupGetter__)                      \
     P(__lookupSetter__)                      \
-    P(Atomics)                               \
```

Comment:
If you want to reorder these, please do it in a separate commit :)

Reply:
ok